src/app/core/store/documents/document.model.ts

```diff
@@ -37,3 +37,9 @@ export interface DocumentModel {
   correlationId?: string;
 
 }
+
+export interface DocumentDataModel {
+  attributeId?: string,
+  name?: string,
+  value?: any
```

Comment:
Why do we store it this way? Have not we agreed to use just `attributeId` -> `value` map? I can see that we now do many unnecessary complicated mappings.

Reply:
I don't think so that there are complicated mapings. It's needed because when we dispatch action for create/update/patch attributeId can be null. So how can I store unitialized data in map? The answer is simple: it is impossible, because key in map can not be null.